adding prompt term (#2231)
Codecademy/docs: content/javascript/concepts/window-prompt/window-prompt.md

---

> Title: 'prompt()'

Reviewer: The folder/file name needs to match this (`promt`)

Also, the window concept isn't merged yet but when it does, this can follow shortly after. Just make sure to create a window folder so the whole path for this changed file looks like `content/javascript/concepts/window/terms/prompt/prompt`

Description: 'Displays a dialog with an optional message asking users to input some text.'
Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Parameters'
  - 'Arguments'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

> The **`prompt()`** method is serviced globally by the `window` object provided by the user's browser. It displays a dialog with an optional message asking users to input some text.

Reviewer: function*

Reviewer: Since we're talking within the scope of window object, we'll refer to it as such

## Syntax

> `prompt()` expects 1 to 2 parameters. The first one is used to display the title of the dialog. The second is used for an optional default value used in case the user does not input any data, otherwise `prompt()` returns null.

Reviewer: Pedagogy: would be helpful to include these parameter explanations after the examples of how this function can be composed

> Below shows some examples of how `prompt()` is composed:

Reviewer: ```suggestion
Below are examples of how `prompt()` can be composed:
```

```pseudo
prompt()

prompt(message)

prompt(message, defaultValue)
```

## Example

> In the following code sample, the browser will automatically stop any running Javascript code until the user clicks on the `OK` button or `cancel` button. If the user clicks on cancel button 'Chihuahuas' will be printed out.

Reviewer: ```suggestion
In the following code sample, the browser will automatically stop any running JavaScript code until the user clicks on the `OK` button or `cancel` button. If the user clicks on the cancel button, `'Chihuahuas'` will be printed out.
```

> ```shell

Reviewer: I think you meant for this to be a javascript block. If you don't want a separate shell block to display output, you can turn this into a codebyte instead

---
Title: 'prompt()'

> Description: 'Displays a dialog with an optional message asking users to input some text'

Reviewer 2: ```suggestion
Description: 'Displays a dialog with an optional message asking users to input some text.'
```

Subjects:
  - 'Web Development'
  - 'Computer Science'
Tags:
  - 'Functions'
  - 'Parameters'
  - 'Arguments'
CatalogContent:
  - 'introduction-to-javascript'
  - 'paths/front-end-engineer-career-path'
---

> **`prompt()`** is a method serviced globally by the window object provided by the user's browser.

Reviewer 2: ```suggestion
The **`prompt()`** method is serviced globally by the `window` object provided by the user's browser. It displays a dialog with an optional message asking users to input some text.
```
The information in the description also needs to be contained in the start of the entry.

## Syntax

> **`prompt()`** expects 1 to 2 parameters, the first one display the title of the dialog, and the second is used for default value that can also be optional and it is used in case that the user does not input any data, otherwise would be null.

Reviewer 2: ```suggestion
`prompt()` expects 1 to 2 parameters. The first one is used to display the title of the dialog. The second is used for an optional default value used in case the user does not input any data, otherwise `prompt()` returns null.
```
Only bold the first occurrence of the term in the entry.

> The provided sample code contains some examples of how `prompt()` is composed:

Reviewer 2: ```suggestion
Below shows some examples of how `prompt()` is composed:
```

```pseudo
prompt()

prompt(message)

prompt(message, defaultValue)
```

## Example

> 

Reviewer 2: Instead of using comments in the code, have an introductory sentence before the code block explaining what it does.
